### DISCUSSION TITLE
A question regarding the power supply using the columns connector

### DISCUSSION THREAD

**Discussion by DK100** [bitfieldlabs/afterglow#12] [Q&A]
Hi! I tried to power the board via the columns connector (Judge Dredd machine) but I it seems that the Arduino doesn't get enough power. The green LED barely lights up (I used a jumper for R67 to connect the columns power), compared to the direct 18V supply (jumper on R67 removed). I might have soldered something wrong, but I wonder, how the power supply using the columns connector should work. I took a look at the schematics and I thought there would be some sort of 18V VCC line in the connector. But there are "only" column lines (compared to a STTNG the JD doesn't use all column lines available).
So when there is no constant 18V line provided by the connector then all columns provide "a portion" of the power when they are activated? If so and the JD doesn't use all column lines, could this be the reason why the Arduino seems to get not enough power?

**Comment by bitfieldlabs** [comment]
There is always only one column line providing +18V on the column connector. These lines are switched every 2 milliseconds. The capacitor C1 is used to smooth the combined lines. When bridging R67 you should use something stronger than just a jumper as all of the boards current (up to 2A) will flow through this connection.
